config/schema.go

```diff
@@ -7,479 +7,922 @@ var schemaDataV1 = `{
   "type": "object",
 
   "patternProperties": {
-    "^[a-zA-Z0-9._-]+$": {
-      "$ref": "#/definitions/service"
-    }
+	"^[a-zA-Z0-9._-]+$": {
```

Comment:
I think we should extract those in `json` files and have `go generate` create this file from it (like we do on `docker/cli`). Not sure if we should do it in the  PR or not

Reply:
I agree, I was hoping someone would mention this. It's definitely annoying to have them in a huge file.

Reply:
I checked out `docker/cli` ([this is it correct?](https://github.com/docker/cli/tree/master/cli/compose/schema)) and noticed that it uses `go-bindata`. I've used that in the past for storing JSON schemas into a binary loading them at run time, so I understand the context. Is that how you foresee doing things in `libcompose`?

If so, I can probably swing that.

Reply:
@cheapRoc yep :angel: